samza-core/src/main/java/org/apache/samza/container/RunLoopFactory.java

```diff
@@ -98,7 +98,7 @@ public Boolean apply(TaskInstance t) {
       log.info("Run loop in asynchronous mode.");
 
       return new AsyncRunLoop(
-        JavaConversions.mapAsJavaMap(taskInstances),
+        JavaConverters.mapAsJavaMapConverter(taskInstances).asJava(),
```

Comment:
`mapAsJavaMap` won't work in scala 2.10 because this method was introduced in scala 2.11. If we want to keep backward compatibility with scala 2.10 we can't use it.